docs/src/app/(public)/docs/[...slug]/page.tsx

```diff
@@ -29,6 +30,7 @@ export default async function Docs({ params }: { params: { slug: string } }) {
           {page.title}
         </h1>
         <div className="flex flex-col gap-4">
+          {/* @ts-expect-error react-server component */}
```

Comment:
I'm a bit bamboozled why this is suddenly throwing a TS error:

<img width="663" alt="image" src="https://github.com/Thinkmill/keystatic/assets/5933897/d74661e2-4130-46f3-8ec8-823aa9c388f0">

Will do some digging, but don't _think_ it should be a blocker

Reply:
Possibly due to the updated react types where promises are valid?

Reply:
> Possibly due to the updated react types where promises are valid?

Good call, my local was out of date.